eng/workloads/workloads.csproj

```diff
@@ -165,28 +158,38 @@
     <MSBuild Projects="@(VisualStudioManifestProjects)" BuildInParallel="true" Properties="SwixBuildTargets=$(SwixBuildTargets);ManifestOutputPath=$(VersionedVisualStudioSetupInsertionPath);OutputPath=$(VersionedVisualStudioSetupInsertionPath)" />
   </Target>
 
-  <Target Name="GenerateVersions" DependsOnTargets="GetAssemblyVersion">
-    <Exec Command="git rev-list --count HEAD"
-          ConsoleToMSBuild="true"
-          Condition=" '$(GitCommitCount)' == '' ">
-      <Output TaskParameter="ConsoleOutput" PropertyName="GitCommitCount" />
-    </Exec>
-
-    <Error Condition=" '$(OfficialBuild)' == 'true' And '$(_PatchNumber)' == '' "
-           Text="_PatchNumber should not be empty in an official build. Check if there were changes in Arcade." />
+  <Target Name="_GetVersionProps">
+    <PropertyGroup>
+      <_MajorVersion>$([System.Version]::Parse('$(AssemblyVersion)').Major)</_MajorVersion>
```

Comment:
Where is this AssemblyVersion property coming from?

Reply:
https://github.com/dotnet/emsdk/blob/7d374c39385b45fbc762bba446bd9baad32a18d5/eng/workloads/workloads.csproj#L53 

The `GetAssemblyVersion` target